### PR TITLE
Add new parameters to support use_annotations

### PR DESCRIPTION
**Pull request by allaway**
I think this is a breaking change; it refers to parameters that don't exist in the current getModelManifest function, so will require users to update to a version of schematic with the changes proposed in PR 456: https://github.com/Sage-Bionetworks/schematic/pull/456

These changes surface the sheet_url and use_annotations parameters from some of the lower level functions for the shiny app. I found through testing that I needed to surface the sheet_url parameter, otherwise getModelManifest returned a manifest_df and not a manifest_url to the shiny app. I am not 100% sure why that started happening after adding the use_annotations parameter.

### CODE REVIEW

**Comment by allaway** [comment]
I think it would be really swell if `use_annotations` T/F was surfaced to the end user with a prettyCheckboxInput, but I don't have time at the moment to add and test that.

File: app.R [hunk 1]
```diff
@@ -379,7 +379,13 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
...
 
 
-      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), template_type, filenames = as.list(filename_list))
+      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), 
+						      template_type, 
+						      filenames = as.list(filename_list),
+						      datasetId = folder_synID,
+						      filenames = as.list(filename_list),
+						      useAnnotations = F,
+						      sheetUrl = T)
```
Comment (sujaypatil96) [inline app.R:388]: Per the new `getModelManifest()` [signature](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/models/metadata.py#L115), we can remove explicit specification of both `useAnnotations` and `sheetUrl`.

**Review by sujaypatil96 — COMMENTED** [review]
Apologies for the late review @allaway. Looks like we can close this PR since the default values of `useAnnotations` and `sheetUrl` are the same as this change?

**Comment by allaway** [comment]
Yep, can definitely close this!

**Comment by allaway** [comment]
Ok, so after some further testing, I found that we do need to explicitly define the `datasetId` parameter if `use_annotations = T` (without it, the annotations don't actually populate), so suggesting this change.

File: app.R [hunk 2]
```diff
@@ -379,7 +379,10 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
       filename_list <- names(file_namedList)
 
 
-      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), template_type, filenames = as.list(filename_list))
+      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type),
```
Comment (milen-sage) [inline app.R:382]: @allaway I might be missing something, but it looks like the changes to the function here do not surface the [changes](https://github.com/Sage-Bionetworks/schematic/pull/456) in schematic? (The latter have been merged btw.)

Comment (allaway) [reply app.R:382]: Hi @milen-sage , I removed those @sujaypatil96's recommendation given the default behavior of schematic (though I still have useAnnotations=T in my local branch). https://github.com/Sage-Bionetworks/data_curator/pull/157#discussion_r624038669

**Comment by sujaypatil96** [comment]
@allaway: I/we changed the base branch that this PR was merging to from `shiny-server-main` to `shiny-server-develop` since most of the development code we are adding lives there. Due to that, there's a resulting merge conflict, since the base branch (refactored) does not have the monolithic `app.R` file anymore. You'll need to make the change you suggested here to `server.R` instead (let me know if you need me to take care of this for you).

Also the main agenda of the PR can be changed from `Add new parameters to support use_annotations` to something like `Add dataset_id to support use of use_annotations`.

**Comment by allaway** [comment]
@sujaypatil96 , done in #173!

**Pull request closed without merging**